src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
In my rewrites it always ends up being a generic container view in place of page, article and block. I then tend to have a distinct menu view and a component view. So... 3 classes instead of 5 and an abstract.